ingestion/functions/parsing/canada/sample_data.csv

```diff
@@ -0,0 +1,2 @@
+"case_id","provincial_case_id","age","sex","health_region","province","country","date_report","report_week","travel_yn","travel_history_country","locally_acquired","case_source","additional_info","additional_source","method_note"
```

Comment:
We have a `travelHistory` field -- it'd be awesome to capture this travel data there. I think the only data we'd be able to record is country.

@attwad -- do we geocode travel history locations? I don't see a `query` field in the API.

Reply:
Seems like this field needs lots of details: dateRange & purpose methods. However, we only have a country.
![image](https://user-images.githubusercontent.com/6234533/94640644-1c692400-0294-11eb-905e-09b0858dd667.png)

Reply:
Yes we do geocode travels now, you can output something like:

```
{
  travelledPrior30Days: true,
  travel: [
    {
      location: { query: "Cuba" }
    }
  ]
}
```

and it will work

Reply:
What will happen if the query is not eligible?  (like "Grand Princess Cruise Ship")